src/zc/buildout/buildout.py

```diff
@@ -384,7 +384,12 @@ def __init__(self, config_file, cloptions,
 
         download_cache = options.get('download-cache')
         extends_cache = options.get('extends-cache')
+
+        if bool_option(options, 'abi-tag-eggs', 'false'):
+            from zc.buildout.pep425tags import get_abi_tag
+            options['eggs-directory'] += '-' + get_abi_tag()
```

Comment:
Hi @jimfulton, nice feature!

@jamadden wrote:
> FWIW, if possible I would prefer to have the api-specific eggs-directory beneath the egg directory I specify (or is default). [...]

I second that, for all the given reasons.

Even better would be if the abi-independent eggs remained in the root `buildout:eggs-directory` to save space. The space saving of sharing eggs between multiple buildouts (when configured at the user home) and buildout parts (at the very least) is one of the best features of buildout that is still missing from `virtualenv` based solutions.

But I guess this would require a much bigger PR, and we could have problems of incompatibility with the generated .pyc files in Python < 3.2 (would there be any such problem in reality?)